### PR TITLE
Added browserext and browserext-native

### PR DESCRIPTION
**Pull request by mikepie1**
W3C Browser Extensions Community Group draft reports

### CODE REVIEW

**Comment by tobie** [comment]
Thanks for the PR. I clarified the [contributing guide](https://github.com/tobie/specref/blob/master/README.md#manual-changes) a bit, but I'd rather keep `./refs/w3c.json` for auto-generated content for now.

File: refs/w3c.json [hunk 1]
```diff
@@ -739,6 +739,27 @@
     "ATAG20-TECHS": {
         "aliasOf": "IMPLEMENTING-ATAG20"
     },
+    "browserext": {
+        "authors": [
+            "Mike Pietraszak"
+        ],
+        "href": "https://browserext.github.io/browserext/",
+        "title": "Browser Extensions",
+        "date": "October 2016",
```
Comment (tobie) [inline refs/w3c.json]: While you're moving files, think you could also use the following instead:

```js
{
  // ...
  "rawDate": "2016-10",
  // ...
}

Comment (mikepie1) [inline refs/w3c.json]: Sounds good. I updated the pull request with these changes

**Comment by tobie** [comment]
Tests are failing because you have both `date` and `rawDate` on a reference.

**Comment by tobie** [comment]
Thanks for your contribution and patience getting it reviewed.

Now live: http://www.specref.org/?q=browserext

**Comment by mikepie1** [comment]
No problem. Thank you for guiding me!

**Pull request merged**